community/sigs/01_intro.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "What is a SIG?"
+---
+
+Eclipse Tractus-X consists of different applications, products, KITs and also hosts other supporting repositories.
+All of these repositories have a very dedicated goal and focus.
+However, there are of course cross-cutting topics, that affect our whole project.
+
+To foster these needs, we decided to form "Special Interest Groups", or "SIGs" in short.
+SIGs 
```

Comment:
Not sure, if that is kind of leftover, orphan word? :)

Reply:
oh that slipped through :) thanks. I think I wanted to start a new sentence, but ran out of ideas how to finish it . Removed